### PR TITLE
Add validations for amp-lightbox-gallery:1.0

### PR DESCRIPTION
**Pull request by caroqliu**
For #32759

### CODE REVIEW

**Review by Gregable — APPROVED** [review]

File: extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      type="a9"
+      data-aax_size="300x250"
+      data-aax_pubname="test123"
+      data-aax_src="302">
+    </amp-ad>
+  </amp-base-carousel>
+
+  <!-- Invalid: missing value for lightbox-thumbnail-id -->
+  <amp-img src="/awesome.png" width="300" height="300" lightbox lightbox-thumbnail-id></amp-img>
+  <!-- Invalid: invalid use of lightbox-exclude -->
```
Comment (samouri) [inline extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html:106]: nit: how is it an invalid use?

**Review by samouri — APPROVED** [review]

File: extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html (hunk 1 above)
Comment (caroqliu) [reply extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html:106]: It looks like `lightbox-exclude` is specific for children of lightboxed carousel elements (as in, lightbox all the elements except the ones with `lightbox-exclude`)

Comment (samouri) [reply extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html:106]: the nit was meant to be to include that detail in the comment!

Comment (caroqliu) [reply extensions/amp-lightbox-gallery/1.0/test/validator-amp-lightbox-gallery.html:106]: oh 😅

**Pull request merged**